numpy/linalg/linalg.py

```diff
@@ -2006,7 +2006,7 @@ def _multi_svd_norm(x, row_axis, col_axis, op):
     """
     if row_axis > col_axis:
         row_axis -= 1
-    y = rollaxis(rollaxis(x, col_axis, x.ndim), row_axis, -1)
+    y = moveaxis(x, [row_axis, col_axis], [-2, -1])
```

Comment:
The commit that introduced this function is 4 years old, a2926149891027fff151d50933405a696384d47c, and the code hasn't been touched since.